frontend/src/metabase/admin/datamodel/components/GuiQueryEditor/GuiQueryEditor.jsx

```diff
@@ -224,9 +231,10 @@ export class GuiQueryEditor extends Component {
 
     return (
       <div
-        className={
-          "GuiBuilder-section GuiBuilder-data flex align-center arrow-right"
-        }
+        className={cx(
+          CS.arrow_right,
```

Comment:
There's not even this class exposed. Please see comments in `frontend/src/metabase/css/core/arrow.module.css`